app/views/facilities/show.html.erb

```diff
@@ -22,5 +22,36 @@
         <div class="text-sm text-caption-light font-medium uppercase mt-2">Code</div>
         <%= @facility.code %>
     </div>
+
+    <div class="facilities__index__container__header">
```

Comment:
Do we still need these facility tasks now that we've implemented TW?

Reply:
No but we are pushing up so everyone has a copy of how the other pages should look so that they can style the other pages. I will be fixing the old classes on locations stuff once this is merged in.